### DISCUSSION TITLE
platform_check.php requiring PHP 8

### DISCUSSION THREAD

**Discussion by Florent-lux** [DirectoryLister/DirectoryLister#759] [General]
Hi! I noticed `app/vendor/composer/platform_check.php` requires PHP 8 in release 3.10, although the site mentions PHP 7.3+. Just wanted to point out the discrepancy. Thanks for the great tool.

**Comment by PHLAK** [comment]
Thanks, I will look into this.

**Comment by PHLAK** [comment]
This should now be fixed with [v3.10.1](https://github.com/DirectoryLister/DirectoryLister/releases/tag/3.10.1).

**Comment by Florent-lux** [comment]
Great, many thanks!
